packages/suite/src/components/suite/NotificationCard.tsx

```diff
@@ -80,46 +92,51 @@ const Body = styled.div`
     justify-content: space-between;
     align-items: center;
     width: 100%;
-    color: ${({ theme }) => theme.TYPE_DARK_GREY};
-    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
-    font-size: ${variables.FONT_SIZE.SMALL};
+    color: ${({ theme }) => theme.textDefault};
+    ${typography.hint}
 `;
 
-const NotificationButton = styled(Button)<{
-    notificationVariant: NotificationCardProps['variant'];
-}>`
-    margin-left: 16px;
-    color: ${({ theme }) => theme.TYPE_WHITE};
-    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
-    font-size: ${variables.FONT_SIZE.SMALL};
-    background: ${({ notificationVariant, theme }) => getMainColor(notificationVariant, theme)};
-    height: 30px;
-
-    &:hover,
-    &:focus,
-    &:active {
-        color: ${({ theme }) => theme.TYPE_WHITE};
-        background: ${({ notificationVariant, theme }) =>
-            getHoverColor(notificationVariant, theme)};
+const CardButton = ({ href, ...props }: ButtonType) => {
+    if (href) {
+        <TrezorLink variant="nostyle" href={href}>
```

Comment:
`TrezorLink` blocks component from moving to `@trezor/components`. Any ideas here?

Reply:
Totally not now, we are not moving yet ;)